Adding CAP eCP Codes

goelalex opened the pull request:
Adding CAP eCP Codes in support of the IHE SDC/eCP on FHIR IG https://build.fhir.org/ig/HL7/ihe-sdc-ecc-on-fhir/

The codes are required to fix the terminology error on the QA page of that IG before it can be approved by TSC for STU1 publication 

CAP and HL7 have a licensing agreement in place for use of the eCPs in the IGs. 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205319149011612

goelalex committed:
Adding CAP eCP Codes

Adding CAP eCP Codes in support of the IHE SDC/eCP on FHIR IG https://build.fhir.org/ig/HL7/ihe-sdc-ecc-on-fhir/

The codes are required to fix the terminology error on the QA page of that IG before it can be approved by TSC for STU1 publication

grahamegrieve reviewed (commented):
This is not acceptable without content in the copyright element - is posting this OK with CAP?

goelalex commented:
Converting to Draft until Copyright from CAP is added.

goelalex committed:
Showing only subset of CAP codes for testing purposes

grahamegrieve commented:
this claims content = complete. Surely that's not all the codes?

grahamegrieve commented:
Also it doesn't have a url or a version

goelalex committed:
Updating version, content, ID, last updated and identifier

goelalex commented:
> Also it doesn't have a url or a version

Added URL, updated the version, added meta.lastupdated, id and identifier on Dec 6, 2023 commit @grahamegrieve

reubendaniels commented:
Hi Alex

Two follow-up queries for you:

**1. CodeSystem.content**
I note that you have specified the value of the content element as `example` which is defined as:

> _A subset of the valid externally defined concepts are included in the code system resource. There is no specific purpose or documented intent other than for illustrative purposes._

Is that consistent with the HL7/CAP licensing arrangement and the IG's intent? 

Or would a value of `fragment` which is defined as:

> _A subset of the code system concepts are included in the code system resource. This is a curated subset released for a specific purpose under the governance of the code system steward, and that the intent, bounds and consequences of the fragmentation are clearly defined in the fragment or the code system documentation. Fragments are also known as partitions._

be more appropriate?

**2. CodeSystem.version**
I note that the version id you have provided is `100004300` which is a bit unusual (we usually have the values in YYYYMMDD or x.y.z formats). Does this correspond to a versioning scheme from CAP? Either way, can you please provide some information regarding the versioning scheme in use?

Thanks

goelalex committed:
updating content to "fragment" and version to "4.3.1.0"

goelalex commented:
@reubendaniels I updated the content to be "fragment" which does make more sense, and I changed the version to "4.000.001" which was the version I got from the team as the version of the codes used in this fragment codesystem - which is the latest

goelalex committed:
corrrecting version from 4.3.1.0 to 4.000.001

grahamegrieve reviewed (approved):
ok

goelalex commented:
@grahamegrieve how long before the codes are available in the publisher for validation?
